docs/proposals/sidecar.md

```diff
@@ -0,0 +1,40 @@
+# Kubelet Executes Commands in Sidecar Containers
```

Comment:
"sidecar" is not the term you want - we've been using that term to mean specifically helper containers run in a pod alongside user containers.  You need a different word.